webservice/app/Providers/EventServiceProvider.php

```diff
@@ -26,20 +26,5 @@ class EventServiceProvider extends ServiceProvider
     public function boot()
     {
         parent::boot();
-
-        Event::listen('tymon.jwt.absent', function () {
```

Comment:
@cristianopacheco Quick question: it is safe to remove those listeners as well as remove the middleware `jwt.auth` from the route group because we're now using provided driver from the TymonDesign version 1. Right?